include/native/parallel_execution_native.h

```diff
@@ -757,7 +757,7 @@ template <typename ... InputIterators, typename Identity,
 auto parallel_execution_native::map_reduce(
     std::tuple<InputIterators...> firsts, 
     std::size_t sequence_size,
-    Identity && identity,
+    Identity &&,
```

Comment:
We should probably open an issue to better understand why identity value is not being used in this version of map/reduce.

Reply:
I have opened issue #334 to handle this separately.

Reply:
Fixed by changing the "child" sequential map_reduces to have the exact same input as the parent